WordPress/src/main/java/org/wordpress/android/ui/JetpackConnectionSource.kt

```diff
@@ -0,0 +1,25 @@
+package org.wordpress.android.ui
+
+enum class JetpackConnectionSource(private val value: String) {
+    NOTIFICATIONS("notifications"), STATS("stats");
+
+    override fun toString(): String {
+        return value
+    }
+
+    companion object {
+        @JvmStatic fun fromString(value: String): JetpackConnectionSource? {
```

Comment:
1) Suggestion (💡): I suggest having the `@JvmStatic` annotation on a separate line.
2) Suggestion (💡): This `fromString(...)` function can be further simplified to:
```
        fun fromString(value: String) = when {
            NOTIFICATIONS.value == value -> NOTIFICATIONS
            STATS.value == value -> STATS
            else -> null
        }
```

Reply:
Ugh, running the code reformat moves everything to a separate line again.

Reply:
🤔 What do you mean Annmarie, we can double check on that by pairing.

Reply:
If you mean that reformatting moves the `@JvmStatic` annotation on the same line, then this is an IDE settings that can be fine tuned.

Reply:
See 57d5ab0

Reply:
👋 @zwarm ! This is probably the wrong commit, right?

Reply:
Also, if you meant that reformatting moves the `when` statement lines, then this is because the `braces` need to be removed as well.